src/Compilers/VisualBasic/Portable/BoundTree/Expression.vb

```diff
@@ -1382,6 +1370,105 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
         End Function
     End Class
 
+    Partial Class BoundFieldOrPropertyInitializer
+        Implements IFieldInitializer, IPropertyInitializer
+
+        Private ReadOnly Property IField As IFieldSymbol Implements IFieldInitializer.Field
```

Comment:
I haven't yet done the editing pass to switch to an InterfaceName_MemberName naming pattern.